cmd/agent/main.go

```diff
@@ -37,6 +37,12 @@ var cmds = []*cli.Command{
 				Name:  "image",
 				Usage: "Specify an full image reference, e.g.: quay.io/some/image:tag",
 			},
+			&cli.StringFlag{Name: "auth-username", Usage: "Username to authenticate to registry"},
```

Comment:
I guess we could expose some env var as well? but this looks good anyway :+1:

Reply:
This should go away if we kind of move the installation stuff over elemental directly and remove it from the agent in the future-ish